src/aics-image-viewer/components/ColorPicker.jsx

```diff
@@ -58,48 +68,56 @@ class ColorPicker extends React.Component {
 
   render() {
     const width = this.props.width || 36;
-    const styles = reactCSS({
-      'default': {
-        color: {
-          width: `${width}px`,
-          height: '14px',
-          borderRadius: '2px',
-          background: `rgba(${ map(this.state.color, ele => (ele))})`
-        },
-        swatch: {
-          padding: '5px',
-          borderRadius: '1px',
-          boxShadow: '0 0 0 1px rgba(0,0,0,.1)',
-          display: 'inline-block',
-          cursor: 'pointer'
-        },
-        popover: {
-          position: 'absolute',
-          zIndex: '9999',
-        },
-        cover: {
-          position: 'fixed',
-          top: '0px',
-          right: '0px',
-          bottom: '0px',
-          left: '0px'
-        }
-      }
-    });
+
+    let popoverDirectionStyle = {};
```

Comment:
const. 
could even be 
```
const popoverDirectionStyle = this.state.openAboveSwatch ? {bottom:"25px"} : {top:"1px"};
```
I suspect, because of the single quotes around the px values, that you need to run `npm run lint`, `npm run typeCheck`, and run prettier (formatter) to do the basic formatting checks.
We should probably see if there's a way to ensure autoformatting on commit or something.  I have "format on save" enabled for the project in vscode settings.

Reply:
Actually I see that the gh-actions did run the lint but I'm surprised it didn't warn about let vs const. I guess volume-viewer warns about this but website-3d-cell-viewer doesn't, in the eslint config ( "prefer-const": ["warn"] ).
I would advocate for adding this but in a separate maintenance pr because it might trigger a whole bunch of warnings.

Reply:
Yep, your way is better on this. Don't know how I missed it in the first place. And eslint throws up ~50 warnings when told to care about const, so definitely an issue for another time.